internals/plan/plan.go

```diff
@@ -825,11 +932,18 @@ func (layer *Layer) Validate() error {
 		}
 	}
 
+	for field, section := range layer.Sections {
+		err := section.Validate()
+		if err != nil {
+			return fmt.Errorf("cannot validate layer section %q: %w", field, err)
```

Comment:
That feels like unnecessary verbosity given the context. Consider the errors above, for example:
```
fmt.Sprintf("plan check %q timeout must not be zero", name),
```
This is the sort of error that a section extension should emit, so there's no need to prefix these and get:

> cannot validate layer section "checks": plan check %q timeout must not be zero

for example.

IOW, this is "our turf", so let's make sure the extensions are emitting pleasant messages.

Reply:
Removed wrapping of the error. Thank you. Resolved.